### ISSUE TITLE
Automod doesn't notify you, that you massage didn't go thru, if you switched accounts  

### ISSUE THREAD

**Issue by Schmopus** [Chatterino/chatterino2#3682]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

If you wave switched accounts since starting Chatterino and post a massage wich is filtered out by Automod the massage "AutoMod: Hey! Your message is being checked by mods and has not been sent " will not appear, even tho the massage was blocked by Automod.

### OS and Chatterino Version

Chatterino 7.3.5 and windows 10

**Comment by Felanbird** [comment]
Duplicate of #1022

**Comment by Felanbird** [comment]
Being worked on currently in #3643
